src/test/java/com/squareup/kotlinpoet/TypeSpecTest.kt

```diff
@@ -236,8 +236,9 @@ class TypeSpecTest {
         .addFun(FunSpec.builder("fooBar")
             .addModifiers(KModifier.PUBLIC, KModifier.ABSTRACT)
             .addAnnotation(AnnotationSpec.builder(headers)
-                .addMember("value", "%S", "Accept: application/json")
-                .addMember("value", "%S", "User-Agent: foobar")
+                .addMemberArrayElement("value", Array<String>::class, "%S",
```

Comment:
For what it's worth, the generic being specified here is lost. We only receive `KClass<Array>` not `KClass<Array<String>>`.